src/cmd/ksh93/tests/jobs.exp

```diff
@@ -0,0 +1,61 @@
+# Interactive tests for jobs, bg, fg and related builtins
+set pid [spawn $ksh]
+expect_prompt
+
+# ======
+# Ctrl-Z should stop current process
+log_test_entry
+send "cat\r"
+sleep 0.5
+send [ctrl Z]
+expect -re ".*Stopped.*cat.*" {
+    puts "Ctrl-z stops cat processs"
+}
+
```

Comment:
@krader1961 Why putting `expect_prompt` here causes tests to fail ?

Reply:
Because your previous regex captures the prompt. Specifically, the trailing `.*`. Replace that wit h`\r\n`. Also, for reasons I haven't figured out the "Stopped" message isn't appearing in the expect buffer when I run this on macOS.